### ISSUE TITLE
Song audio stops working

### ISSUE THREAD

**Issue by olav35** [bui/taiko-web#293]
**Problem**
After playing the song "Onpu no Uta" once, subsequent plays are missing the song audio. Other songs can be played fine.

**Steps to reproduce**
1. Open incognito chromium
2. Play Onpu nu Uta on extreme

Now if you try opening, Onpu nu Uta again there will be no audio.

---
Chromium Version 87.0.4280.66 (Official Build) snap (64-bit)
```
$ uname -a
Linux Shiina 5.4.0-56-generic #62-Ubuntu SMP Mon Nov 23 19:20:19 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by olav35** [comment]
Related to https://github.com/bui/taiko-web/issues/281, although this issue(#293) is not on a custom song.

**Comment by LoveEevee** [comment]
Fixed in #294
